src/pyspiffe/workloadapi/workload_api_client.py

```diff
@@ -0,0 +1,66 @@
+from abc import ABC, abstractmethod
+from pyspiffe.bundle.x509_bundle.x509_bundle_set import X509BundleSet
+from pyspiffe.bundle.jwt_bundle.jwt_bundle_set import JwtBundleSet
+from pyspiffe.svid.x509_svid import X509Svid
+from pyspiffe.svid.jwt_svid import JwtSvid
+
+
+class WorkloadApiClient(ABC):
```

Comment:
1) Document the methods `fetch_` describing that they are a `one-shot` call. 
2) The `watch_` methods are missing: e.g. `watch_x509_svids`, `watch_jwt_bundles`

Reply:
The `watch` methods are missing be design since we are only implementing synchronous calls in py-spiffe at this time. I didn't think adding the `one-shot` text improved clarity as those are the only call forms for now. I'm happy to add (1) in, but I think we should leave (2) off since async calls are not currently scoped for the library.